spec/acceptance/validate_metadata_spec.rb

```diff
@@ -100,7 +100,7 @@
       end
 
       its(:stdout) do
-        is_expected.to have_junit_testcase.in_testsuite('metadata-json-lint').with_attributes(
+        is_expected.not_to have_junit_testcase.in_testsuite('metadata-json-lint').with_attributes(
           'classname' => 'metadata-json-lint.dependencies',
           'name'      => 'broken.json',
         ).that_failed
```

Comment:
.that_failed can be removed now that we're asserting the testcase does not exist